std_msgs/String map_name
---
grid_map_msgs/GridMap grid_map